als-server/jvm/src/main/scala/org/mulesoft/als/server/acv/CustomDiagnosticReportBuilder.scala

```diff
@@ -33,7 +33,7 @@ object CustomDiagnosticReportBuilder {
   def toLocation(location: OpaLocation): Location =
     new Location(location.location.getOrElse(""), location.range.map(toRange).getOrElse(rangeZero))
   def toRange(range: OpaRange): Range             = new Range(toPosition(range.start), toPosition(range.end))
-  def toPosition(position: OpaPosition): Position = new Position(position.line, position.column)
+  def toPosition(position: OpaPosition): Position = new Position(position.line - 1, position.column)
```

Comment:
Maybe we can add a `position.line != 0 ? position.line -1 : position.line` in case amf change or fix